lib/Core/Memory.cpp

```diff
@@ -211,10 +211,10 @@ void ObjectState::flushToConcreteStore(TimingSolver *solver,
 
 void ObjectState::makeConcrete() {
   delete concreteMask;
-  delete flushMask;
+  delete unflushedMask;
   delete[] knownSymbolics;
   concreteMask = 0;
-  flushMask = 0;
+  unflushedMask = 0;
```

Comment:
`nullptr` is the correct type here.

Reply:
I guess I could do this, it was just a rename here.